tests/io/v3/vm/agents/test_agents_api.py

```diff
@@ -0,0 +1,184 @@
+'''
+Test Agents
+'''
+import re
+from uuid import UUID
+
+import pytest
+import responses
+
+BASE_URL = 'https://cloud.tenable.com/api/v3/agents'
+
+
+@responses.activate
+def test_details(api):
+    '''
+    Test case for agents details method
+    '''
+    test_response = {
+        'id': 'd59d1f5b-f775-4061-9e36-fae22ab7518f2596d192e3cf57f8',
+        'name': 'DESKTOP-PSNDJQ6',
+        'platform': 'WINDOWS',
+        'distro': 'win-x86-64',
+        'ip': '192.0.2.57',
+        'last_scanned': 1477011651,
+        'plugin_feed_id': '0',
+        'linked_on': 1468619962,
+        'status': 'off',
+        'groups': [
+            {
+                'name': 'CodyAgents', 'id': 8
+            },
+            {
+                'name': 'Agent Group A', 'id': 3316
+            }
+        ]
+    }
+    agent_id: UUID = 'd59d1f5b-f775-4061-9e36-fae22ab7518f2596d192e3cf57f8'
+    responses.add(
+        responses.GET,
+        re.compile(f'{BASE_URL}/{agent_id}'),
+        json=test_response
+    )
+    res = api.v3.vm.agents.details(agent_id)
+    assert isinstance(res, dict)
```

Comment:
Can you please add one more assert condition to validate the response similar to other test cases?

Reply:
added new asserts.